projects/ng2-date-picker/src/lib/date-picker/date-picker.component.ts

```diff
@@ -93,13 +93,16 @@ export class DatePickerComponent implements OnChanges,
   @Input() maxDate: SingleCalendarValue;
   @Input() minTime: SingleCalendarValue;
   @Input() maxTime: SingleCalendarValue;
+  @Input() dateRangePicker: boolean = false;
+  @Input() dateRange: IDateRange;
```

Comment:
The range value should be set through the controller input - not as an additional Input of the component.

Reply:
I thought that user can be set default values. This is not work now. Should I remove this?